src/components/LayerKey.vue

```diff
@@ -83,9 +84,11 @@ export default {
     blur() {
       this.startListening();
       this.setSelected(undefined);
+      this.hasFocus = false;
```

Comment:
why does this key need focus and none of the others?

Reply:
The core part of how this works is that the input needed to overflow outside the bounds of the key, to do this I used the [pseudo selector :focus](https://github.com/qmk/qmk_configurator/pull/710/files/2b76eda2f5dd6c9eb646fb5b37b32946f0d054d5#diff-b86aa89eb58031c905f46ced082fdd9fR604) to set the input to absolute. Because of this I needed to make sure the titles didnt move from their position so by adding this I could add padding to the bottom of the key title to give the illusion that it didnt move. Without this the title "Any" disappears because each key is display:flex;

I do agree this is a little out of the box but I also wanted to approach this without refactoring any other components.

Reply:
OK, thanks for eli5. this makes a lot more sense now.